src/licensedcode/data/rules/gpl-3.0_20.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - gpl-3.0

```

Comment:
This is not correct: there should be two licenses: gpl-3.0 plus the other one which is an exception of sorts
This should be a template rul too.